tests/test_cases/test_cocotb/test_handle.py

```diff
@@ -34,8 +34,11 @@ def test_lessthan_raises_error(dut):
 
 
 @cocotb.test()
-def test_bad_attr(dut):
-    yield cocotb.triggers.NullTrigger()
+async def test_bad_attr(dut):
+
+    with assert_raises(AttributeError):
+        fake_signal = dut.fake_signal
+
```

Comment:
This is testing the exact same thing below. You probably could delete one or the other.

Reply:
One is testing a heirarchy object, the other an attribute of a nested signal object. I think having both tests is probably a good thing.

Reply:
I was not quite sure, because the other one goes down two levels, I didn't check the RTL if this is supposed to test some record/interface access or so.

What do you prefer, keep the old or the new one? (if, I would vote for new, because apparantely assert_raises should be the new standard.)

Reply:
> One is testing a hierarchy object, the other an attribute of a nested signal object. I think having both tests is probably a good thing.

same thought same time :D

Reply:
I see. Yeah it's appropriate to leave it.